cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -44,6 +44,14 @@
      */
     public static class Push extends ReplicatorBuilder<Datastore, URI, Push> {
 
+        private int changeLimitPerBatch = 500;
+
+        private int batchLimitPerRun = 100;
+
+        private int bulkInsertSize = 10;
+
```

Comment:
@alfinkel They're not constants - I'm not really sure what you mean. They are configurable values which need to have a default in case the user sets them. The user may want the push and pull values to be different and if some of them have the same default values that's entirely coincidental. I know that some people have an allergy to 'magic numbers' but I'm not convinced about having separate constants in a case like this.

@rhyshort  That would entail a loss of functionality. It's quite feasible (for instance) that for some couch installations a user may want a huge changes limit but that wouldn't be appropriate for the local SQLite database. Just because they have the same names doesn't mean they do the same thing.